express/code/blocks/toc-seo/toc-seo.js

```diff
@@ -390,21 +390,6 @@ function updateActiveTOCLink(toc) {
       }
     }
   }
-
-  // Remove active class from all links
```

Comment:
We removed this **activeHeader**, but all the calculation on updateActiveTOCLink seems to be still here? Is this intended or am I missing something? 

If we we are not updating the UI on activeHeader anymore, then we may want to exit much earlier.